GVFS/GVFS.Common/Git/DiffTreeResult.cs

```diff
@@ -82,6 +83,7 @@ public static DiffTreeResult ParseFromDiffTreeLine(string line, string repoRoot)
             DiffTreeResult result = new DiffTreeResult();
             result.SourceIsDirectory = ValidTreeModes.Contains(parts[0]);
             result.TargetIsDirectory = ValidTreeModes.Contains(parts[1]);
+            result.TargetIsSymLink = parts[1] == "120000";

```

Comment:
Minor, there is no need to check `parts[1]` again if `result.TargetIsDirectory` is `true`